docs/reference/cluster/stats.asciidoc

```diff
@@ -38,6 +38,278 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=flat-settings]
 
 include::{docdir}/rest-api/common-parms.asciidoc[tag=timeoutparms]
 
+[[cluster-stats-api-response-body]]
+==== {api-response-body-title}
+
+`_nodes.total`::
+(integer)
+Total number of nodes in the cluster.
+
+`_nodes.successful`::
+(integer)
+Number of nodes that started succesfully.
+
+`_nodes.failed`::
+(integer)
+Number of nodes that failed to start.
+
+`cluster_name`::
+(string)
+Name of the cluster.
+
+`cluster_uuid`::
+(string)
+Unique identifier for the cluster.
+
+`timestamp`::
+(https://en.wikipedia.org/wiki/Unix_time[Unix timestamp])
+Last time the cluster statistics were refreshed.
+
+`status`::
+(string)
+Health status of the cluster based on the state of its primary and replica
+shards. Returned values include:
+
+  `green`:::
+  All primary shard and replica shards are assigned. There are at least two
+  copies of all shards.
+
+  `yellow`:::
+  All primary shards are assigned, but one or more replica shards are
+  unassigned. If a node in the cluster fails, data could be lost.
+
+  `red`:::
+  One or more primary shards are unassigned. This can occur briefly during
+  cluster startup as primary shards are assigned during cluster startup. At
+  other times, this status may indicate data loss.
```

Comment:
Although the last sentence is true I suggest dropping it. From time to time we see users confusing data loss with `red` health and asking us to change the definition of `red` health to mean something other than data loss since it's normally temporary (a node or two are offline).

Reply:
Removed with 798a69315f0.